os.go

```diff
@@ -66,10 +66,14 @@ func init() {
 	if err != nil {
 		log.Printf("user: %s", err)
```

Comment:
Could you add a comment here, something along the lines:

// When the user is not in `/etc/passwd` (for e.g. LDAP) and `CGO_ENABLED=1` in `go env`,
// the cgo implementation of `user.Current()` fails even when HOME and USER are set.

Reply:
Done !